odc/stats/plugins/lc_fc_wo_a0.py

```diff
@@ -0,0 +1,206 @@
+"""
+Plugin of Module A0 in LandCover PipeLine
```

Comment:
Hi @emmaai, is it worth renaming this file to be called something based on the output bands - thinking maybe users will not know what module a0 refers to. Alternatively maybe adding the architecture diagram to the PR and a reference to the output of module

Reply:
Maybe  `lc_fc_wo_stats.py` ?